styx-scheduler-service/src/main/java/com/spotify/styx/docker/KubernetesDockerRunner.java

```diff
@@ -185,15 +187,18 @@
         register(closer, Objects.requireNonNull(scheduledExecutor), "kubernetes-scheduled-executor");
     this.executor = currentContextExecutorService(
         register(closer, new ForkJoinPool(K8S_POD_PROCESSING_THREADS), "kubernetes-executor"));
+    this.executionEnvVars = Objects.requireNonNull(executionEnvVars);
```

Comment:
```suggestion
    this.executionEnvVars = Objects.requireNonNull(executionEnvVars, "executionEnvVars");
```
we get a more descriptive error message this way